certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Baron Blue",
+    "description": "Baron Blue allows you to perform complex operations in only one action. Convert JSON, XML, CSV, XLS, XLSX, and YAML between themselves in any combination. Convert to and from Excel Dates, find information in strings, find content-types and extensions, convert IP addresses (range, CIDR), and much more. New actions come monthly, and if you\u2019re a part of the manueltgomes.com/club, you\u2019ll get the connector for free, so be sure to check it out.",
+    "version": "1.0",
+    "contact": {
+      "name": "Manuel T Gomes Club Support",
+      "url": "https://manueltgomes.com/support",
```

Comment:
@mtgomes looks like this page doesn't exist. I see a message that this page was not found. Could you please check if this URL is the right one?

Reply:
@mtgomes Please provide a valid URL